### PR TITLE
[EN-5989] Updated TruSTAR's default values

### PR DESCRIPTION
**Pull request by marcos-vivar**
### `Resolves`:
- [EN-5989](https://trustar.atlassian.net/browse/EN-5989) **Update TruSTAR's enums default values**

### `Brief Description`:

- **WHAT**: Update TruSTAR's enums with default values.

- **HOW**: Update their name and values and add new ones.

- **WHY**: In order to match our currently supported IOCs types and parameterize default URLs.

### CODE REVIEW

File: trustar/trustar_enums.py [hunk 1]
```diff
@@ -10,30 +10,39 @@ def members(cls):
...
+    
     CORA_MALWARE = "CORA_MALWARE"
-    THREAT_ACTOR = "THREAT_ACTOR"
     CVE = "CVE"
+    MALWARE = "MALWARE"
     MITRE_TACTIC = "MITRE_TACTIC"
+    THREAT_ACTOR = "THREAT_ACTOR"
+
+
+class TruSTARURLs(TSEnum):
```
Comment (YaPeL) [inline trustar/trustar_enums.py:44]: TruStarUrls

File: trustar/trustar.py [hunk 2]
```diff
@@ -12,9 +13,9 @@
 class TruStar:
 
     DEFAULTS = {
-        "auth_endpoint": "https://api.trustar.co/oauth/token",
-        "api_endpoint": "https://api.trustar.co/api/2.0",
-        "station": "https://station.trustar.co",
+        "auth_endpoint": TruSTARURLs.AUTH_TOKEN,
```
Comment (YaPeL) [inline trustar/trustar.py:16]: TruSTARURLs.AUTH_TOKEN.name

File: trustar/trustar.py [hunk 3]
```diff
@@ -12,9 +13,9 @@
...
+        "api_endpoint": TruSTARURLs.API,
```
Comment (YaPeL) [inline trustar/trustar.py:17]: TruSTARURLs.API.name

File: trustar/trustar.py [hunk 4]
```diff
@@ -12,9 +13,9 @@
...
+        "station": TruSTARURLs.STATION,
```
Comment (YaPeL) [inline trustar/trustar.py:18]: TruSTARURLs.STATION.name

**Review by YaPeL — CHANGES_REQUESTED** [review]

**Review by YaPeL — APPROVED** [review]

**Review by hernantain — APPROVED** [review]
LGTM 👍

**Pull request merged**